methods/incidentReports.coffee

```diff
@@ -21,15 +21,22 @@ Meteor.methods
     return newId
 
   # similar to editIncidentReport, but allows you to set a single field without changing any other existing fields.
-  updateIncidentReport: (incident) ->
+  updateIncidentReport: (incident, fieldsToRemove=[]) ->
     _id = incident._id
     delete incident._id
     user = Meteor.user()
     if not Roles.userIsInRole(user._id, ['admin'])
       throw new Meteor.Error("auth", "User does not have permission to edit incidents")
     incident.modifiedByUserId = user._id
     incident.modifiedByUserName = user.profile.name
-    res = Incidents.update({_id: _id}, {$set: incident})
+    updateOperators =
+      $set: incident
+    if fieldsToRemove.length
+      fieldsToUnset = {}
+      fieldsToRemove.forEach (field) ->
```

Comment:
We should limit the removable fields to only cases, deaths and specify so things like modification dates and users can't be altered. Determining the fields to remove in the method might work better than passing them in since we won't need to worry about bad data and it the field removal logic won't need to be repeated in other places where the incident type is altered.